tools/SyncProjects.ps1

```diff
@@ -163,6 +163,10 @@ $($issue.Summary)
       }
       # There's probably more than this for in progress...
       if ($Issue.Status -eq 'In progress') { $hash['expectedColumn'] = 'In progress'}
+      if ($Issue.Status -eq 'Ready for merge') { $hash['expectedColumn'] = 'In progress'}
+      if ($Issue.Status -eq 'Ready for test') { $hash['expectedColumn'] = 'In progress'}
+      if ($Issue.Status -eq 'Ready for ci') { $hash['expectedColumn'] = 'In progress'}
+      if ($Issue.Status -eq 'Ready for review') { $hash['expectedColumn'] = 'In progress'}
       if ($Issue.Status -eq 'Closed') { $hash['expectedColumn'] = 'Done'}
       if ($Issue.Status -eq 'Resolved') { $hash['expectedColumn'] = 'Done'}
       if ($Issue.Status -eq 'Completed') { $hash['expectedColumn'] = 'Done'}
```

Comment:
This block of conditionals should probably just be turned into a switch statement
```powershell
$hash['expectedColumn'] = switch($Issue.Status)
{
  'In progress' { 'In progress' }
  'Ready for merge' { 'In progress' }
  # &c
}
```

Reply:
`switch` doesn't return a value, it executes a block